### PR TITLE
Modal Bottom Sheet implemented

### PR DESCRIPTION
**Pull request by yurizanini**
Now merging into Scheduler.

### CODE REVIEW

File: lib/Views/school_login.dart [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import 'dart:async';
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide ModalBottomSheetRoute;
```
Comment (sapoepsilon) [inline lib/Views/school_login.dart]: Why are you naming the import?

Comment (yurizanini) [reply lib/Views/school_login.dart]: This import has a conflict with the Modal Bottom Sheet library and I had to use hide with that import to work. I found this solution in Stack Overflow.
Here is the error before that change:
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/modal_bottom_sheet-2.1.2/lib/src/material_with_modal_page_route.dart:4:1: Error:
'ModalBottomSheetRoute' is imported from both 'package:flutter/src/material/bottom_sheet.dart' and
'package:modal_bottom_sheet/src/bottom_sheet_route.dart'.

Comment (yurizanini) [reply lib/Views/school_login.dart]: Actually, I just reviewed this again, I imported a new version of the library that resolves this conflict. We can actually discard that change.

Comment (sapoepsilon) [reply lib/Views/school_login.dart]: Ok, just commit the new version, and then looks good to merge

**Review by sapoepsilon — APPROVED** [review]

**Pull request merged**